docs/index.md

```diff
@@ -258,3 +258,15 @@ To get started, see the [Provider Example Configs](https://github.com/couchbasec
 
   Manage Storage Auto-expansion for Azure Clusters.
 
+* [Create Audit Log Export Job in Capella](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/tree/main/examples/audit_log_export)
+
+  Create Audit Log Export Jobs.
```

Comment:
the word job is internal and should be abstracted... so remove it

Reply:
@Talina06 - we refer to jobs even in our public API. That's fine to say that this is an export job. If we want another term, we should think about that and change consistently. Developers / Devops will understand what this entails so would not be concerned